BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsClient.kt

```diff
@@ -173,7 +189,7 @@ internal class AnalyticsClient @VisibleForTesting constructor(
                 data = analyticsRequest.toString(),
                 configuration = null,
                 authorization = authorization,
-                callback = HttpNoResponse()
+                callback = HttpNoTimingResponse()
```

Comment:
I think we can make callback `null` by default. On iOS I think it's needed because we're using generics. Here we can make it optional and guard it's invocation with [an optional unwrap operator](https://stackoverflow.com/a/48185699).

Reply:
Ready, thanks for your comments and sharing documentation [741e0aa](https://github.com/braintree/braintree_android/pull/1032/commits/741e0aa71b124b3dc3f355a4c4b35ec0d56183c3)